scripts/ci/in_container/_in_container_utils.sh

```diff
@@ -74,6 +75,7 @@ function in_container_cleanup_pycache() {
     set +o pipefail
     sudo find . \
         -path "./airflow/www/node_modules" -prune -o \
+        -path "./airflow/www_rbac/node_modules" -prune -o \
```

Comment:
should not we delete the whole directory?

Reply:
No need. Leaving the directory is nice f you keep switching between v1-10 and master because then the node_modules will stay there and will not need to be re-initialized.

Reply:
It's just nicer for someone like me or ash who switch continuously between master and v1-10